src/Shopsys/ShopBundle/Resources/translations/validators.de.po

```diff
@@ -2,155 +2,151 @@ msgid ""
 
 msgid "Please enter first name of contact person"
-msgstr "(de) Please enter first name of contact person"
+msgstr "Bitte geben Sie den Vornamen der Kontaktperson ein"
 
 msgid "Please enter full name"
-msgstr "(de) Please enter full name"
+msgstr "Bitte geben Sie den vollständigen Namen ein"
 
 msgid "Please enter identification number"
-msgstr "(de) Please enter identification number"
+msgstr "Bitte geben Sie die Identifikationsnummer ein"
 
 msgid "Please enter last name"
-msgstr "(de) Please enter last name"
+msgstr "Bitte geben Sie den Nachnamen ein"
 
 msgid "Please enter last name of contact person"
-msgstr "(de) Please enter last name of contact person"
+msgstr "Bitte geben Sie den Nachnamen der Kontaktperson ein"
 
 msgid "Please enter password"
-msgstr "(de) Please enter password"
+msgstr "Bitte Passwort eingeben"
 
 msgid "Please enter price in correct format (positive number with decimal separator)"
-msgstr "(de) Please enter price in correct format (positive number with decimal separator)"
+msgstr "Bitte geben Sie den Preis im richtigen Format ein (positive Zahl mit Dezimaltrennzeichen)"
 
 msgid "Please enter quantity"
-msgstr "(de) Please enter quantity"
+msgstr "Bitte geben Sie die Menge ein"
 
 msgid "Please enter street"
-msgstr "(de) Please enter street"
+msgstr "Bitte geben Sie die Straße ein"
 
 msgid "Please enter telephone number"
-msgstr "(de) Please enter telephone number"
+msgstr "Bitte geben Sie die Telefonnummer ein"
 
 msgid "Please enter valid e-mail"
-msgstr "(de) Please enter valid e-mail"
+msgstr "Bitte geben Sie eine gültige E-Mail Adresse ein"
 
 msgid "Please enter valid quantity"
-msgstr "(de) Please enter valid quantity"
+msgstr "Bitte geben Sie eine gültige Menge ein"
 
 msgid "Please enter zip code"
-msgstr "(de) Please enter zip code"
+msgstr "Bitte geben Sie die Postleitzahl ein"
 
 msgid "Price must be greater or equal to zero"
-msgstr "(de) Price must be greater or equal to zero"
+msgstr "Der Preis muss größer als Null oder gleich Null sein"
 
 msgid "Quantity must be greater than {{ compared_value }}"
-msgstr "(de) Quantity must be greater than {{ compared_value }}"
+msgstr "Die Menge muss größer als {{Vergleichswert }}} sein"
 
 msgid "Street name cannot be longer than {{ limit }} characters"
-msgstr "(de) Street name cannot be longer than {{ limit }} characters"
+msgstr "Der Straßenname darf nicht länger als {{ limit }}} Zeichen sein"
 
 msgid "Tax number cannot be longer than {{ limit }} characters"
-msgstr "(de) Tax number cannot be longer than {{ limit }} characters"
+msgstr "Die Steuernummer darf nicht länger als {{{ limit }}} Zeichen sein"
 
 msgid "Telephone number cannot be longer than {{ limit }} characters"
-msgstr "(de) Telephone number cannot be longer than {{ limit }} characters"
+msgstr "Die Telefonnummer darf nicht länger als {{{ limit }}} Zeichen sein"
 
 msgid "This e-mail is already registered"
-msgstr "(de) This e-mail is already registered"
-
-msgid "This field must be empty"
```

Comment:
The message shouldn't have been removed

Reply:
Or maybe it's OK for the same reason as in https://github.com/shopsys/demoshop/pull/71#discussion_r317463865 - I would rerun `php phing translations-dump` to be sure that nothing was removed accidentally.

Reply:
This translation was deleted earlier. Dump is clean.